source/_integrations/salus.markdown

```diff
@@ -0,0 +1,126 @@
+---
+title: Salus
+description: Instructions on how to integrate your IT500 Salus thermostats within Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2021.4
+ha_iot_class: Cloud Polling
+ha_domain: salus
+ha_config_flow: true
+ha_codeowners:
+  - '@cipacda'
+---
+
+The `salus` integration lets control your connected [IT500 Salus](https://salus-it500.com/public/login.php) thermostat to control your home temperature.
+
+There is currently support for the following device types within Home Assistant:
+
+- Climate
+
+{% include integrations/config_flow.md %}
+
+As a first step you will need to enter your credentials to login into the [Salus account](https://salus-it500.com/public/login.php), and on the second step you will be able to select the device you want to configure from a list of all devices in that account.
+
+Alternatively, add the following information to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+salus:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+  device: 12345
+```
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1:

Reply:
I will remove this from the documentation, but the integration is actually implemented to use config flows, see this from the PR: https://github.com/home-assistant/core/pull/48103/files#diff-aa1a43c9912e7b43e936b45e78ddc8eda11feb31bfdbd3409cc4a9c6dfa51276

I thought it was useful to add the configuration.yaml example as well in the documentation, but I did not use this method as well during implementation.

Please keep this PR open - should I just remove this option frome the documentation?

Reply:
It is implemented in the codebase, hence my comment.

![image](https://user-images.githubusercontent.com/195327/111840719-2a236c00-88fd-11eb-87af-065fe8977c9a.png)

Reply:
Ah, that bit. Fair point, sorry. 

It is my first integration and that would be something left out from when I started from another integration, sorry. That bit can just be removed AFAICT. I assume I can remove that bit since the entire config_flow.py implementation. Can we please leave this PR open and will have that code updated and this documentation on Monday?